pkg/controller/installation/prometheusRules.go

```diff
@@ -62,6 +62,23 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 					},
 				},
 			},
+			{
+				AlertName: "rhmi-upgrade-alerts",
+				Namespace: "openshift-monitoring",
```

Comment:
I would like this namepsace to created as a common constant so that it is easier to maintain and make changes.

Reply:
Thanks for the suggestion! The namespace was already declared as a private constant in the monitoring reconciler, so I just exported it and used that one

Reply:
Thanks. It makes easier to make changes and maintain it and eliminates the most common "typo" errors.